### DISCUSSION TITLE
Multiple Test specs

### DISCUSSION THREAD

**Discussion by bhavya999999** [XRM-OSS/D365-UI-Test#51] [General]
How to run multiple test specs

**Comment by DigitalFlow** [comment]
Hi @bhavya999999,

if you start with the [D365-UI-Test Jest starter](https://github.com/XRM-OSS/D365-UI-Test-Jest-Demo), all .ts files in the `spec` folder should be discovered and executed automatically.

Kind regards,
Florian
